product_category_active/tests/test_product_category_archive.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
```

Comment:
rename tests/test_product_category_**archive**.py to tests/test_product_category_**active**.py